docs/docs/latest/topics/using/edgectl/edge-control.md

```diff
@@ -6,15 +6,16 @@ If you are a developer working on a service that depends on other in-cluster ser
 
 When you want to test your service with traffic from the cluster, use `edgectl intercept` to designate a subset of requests for this service to be redirected to your laptop. You can use those requests to test and debug your local copy of the service running. All other requests will go to the existing service running in the cluster without disruption.
```

Comment:
"to test and debug your local copy of the service ~running~."

I know, not in your changeset...